packages/legend-application-studio/src/stores/graphModifier/DSL_Mapping_GraphModifierHelper.ts

```diff
@@ -97,6 +97,17 @@ export const mapping_setPropertyMappings = action(
     );
   },
 );
+
+export const mapping_deletePropertyMappings = action(
```

Comment:
--> `instanceSetImpl_deletePropertyMapping()`. Also, let's update the one above to `instanceSetImplementation_...`